### ISSUE TITLE
rake apartment:create fails to create tenants when Rails environment is not loaded first

### ISSUE THREAD

**Issue by abhchand** [influitive/apartment#468]
I'm using 
  - Postgres 9.6.1
  - Rails 5.1.0
  - apartment 1.2.0
  - Ruby 2.2.2

My apartnemnt configuration defines 3 tenants:

```ruby
# config/initializers/apartment.rb

ALL_TENANTS = %w(venus earth mars)

Apartment.configure do |config|
  config.tenant_names = ALL_TENANTS
end
```

When I initialize my app for the first time, I create my tenants. 

I'd like to use the built-in apartment rake task - `apartment:create` - to do this. It's defined in [apartment.rake](https://github.com/influitive/apartment/blob/development/lib/tasks/apartment.rake) as:

``` ruby
desc "Create all tenants"
task :create do
  tenants.each do |tenant|
    begin
      puts("Creating #{tenant} tenant")
      Apartment::Tenant.create(tenant)
    rescue Apartment::TenantExists => e
      puts e.message
    end
  end
end
```

However I get

```
Migrating venus tenant
One of the following schema(s) is invalid: "venus" "public"
Migrating earth tenant
One of the following schema(s) is invalid: "earth" "public"
Migrating mars tenant
One of the following schema(s) is invalid: "mars" "public"
Creating venus tenant
rake aborted!
NoMethodError: undefined method `quietly' for main:Object
/Users/abhchand/.rvm/gems/ruby-2.2.2@gb/gems/apartment-1.2.0/lib/tasks/apartment.rake:10:in `block (3 levels) in <top (required)>'
/Users/abhchand/.rvm/gems/ruby-2.2.2@gb/gems/apartment-1.2.0/lib/tasks/apartment.rake:7:in `each'
/Users/abhchand/.rvm/gems/ruby-2.2.2@gb/gems/apartment-1.2.0/lib/tasks/apartment.rake:7:in `block (2 levels) in <top (required)>'
/Users/abhchand/.rvm/gems/ruby-2.2.2@gb/gems/rake-12.0.0/exe/rake:27:in `<top (required)>'
/Users/abhchand/.rvm/gems/ruby-2.2.2@gb/bin/ruby_executable_hooks:15:in `eval'
/Users/abhchand/.rvm/gems/ruby-2.2.2@gb/bin/ruby_executable_hooks:15:in `<main>'
Tasks: TOP => apartment:create
(See full trace by running task with --trace)
```

Not sure exactly what it's stumbling on, but I found that if I override the rake task and load my `:environment` first, it works just fine. 

```diff
namespace :apartment do
  desc "Create all tenants"
- task :create
+ task :create_tenants => :environment do
-   tenants.each do |tenant|
+   Apartment::tenant_names.each do |tenant|
      begin
        puts("Creating #{tenant} tenant")
        Apartment::Tenant.create(tenant)
      rescue Apartment::TenantExists => e
        puts e.message
      end
    end
  end
end
```

Is this something specific to my setup, or do the apartment rake tasks need to load the rails environment as a dependency?

**Comment by meganemura** [comment]
@abhchand 
In case you are using apartment 1.2.0 via rubygems, it has issue in [`apartment:create` task](https://github.com/influitive/apartment/blob/v1.2.0/lib/tasks/apartment.rake#L5-L15) which was resolved in #459. But, newer version is not released yet.

Simple workaround is to define your own rake task or script to run `Apartment::Tenant.create` with each of `tenant_names` IMHO.

**Comment by abhchand** [comment]
@meganemura - Thank you!

That makes sense to me - my version of apartment was using `task create: 'db:migrate' do`, which is why it seemed to run migrate beforehand. 

Will look forward to the new version, and use my own version of the rake task until then.
